requirements.test.txt

```diff
@@ -12,3 +12,4 @@ retry==0.9.2
 types-retry==0.9.9
 jsonpath-ng==1.5.3
 pytest-asyncio==0.19.0
+python-dateutil>=2.8.0
```

Comment:
do we not need types-python-dateutil?

Reply:
I'll double check.

Reply:
I checked and python-dateutil requires types-python-dateutil.